handlers.go

```diff
@@ -50,35 +63,30 @@ func signupSubmit(w http.ResponseWriter, r *http.Request) {
 	// TODO(@FaaizMemonPurdue): Add API call timeouts.
 	err := matcha.database.AddUser(username, email, password)
```

Comment:
I think we should return id,err when return AddUser function

Reply:
yess! you got this!